izpack-panel/src/main/java/com/izforge/izpack/panels/path/PathSelectionPanel.java

```diff
@@ -117,24 +124,25 @@ protected void createLayout()
         IzPanelConstraints ipc = IzPanelLayout.getDefaultConstraint(TEXT_CONSTRAINT);
         // The text field should be stretched.
         ipc.setXStretch(1.0);
-        textField = new JTextField(installData.getInstallPath(), 50);
+        textField = new JTextField(50);
```

Comment:
Don't we loose the initial value from installData?

Reply:
It gets handled by the calling panel in panelActivate method (which uses setPath of PathSelectionPanel. So, it's not a problem.